app/workers/articles/enrich_image_attributes_worker.rb

```diff
@@ -2,7 +2,7 @@ module Articles
   class EnrichImageAttributesWorker
     include Sidekiq::Job
 
-    sidekiq_options queue: :medium_priority, retry: 5, lock: :until_executing
+    sidekiq_options queue: :high_priority, retry: 5, lock: :until_executing
```

Comment:
Are there any scaling/load concerns we should note about switching the queue for this, or is `high_priority` already plenty equipped for this? (do we even do any splitting of queues onto different machines, or is this just a scheduling change on the same box?)

These questions are more for my own learning than to have any strict bearing on the outcome of this PR.

Reply:
@klardotsh I don't think this should be a problem given that this is a relatively infrequent task triggered by direct user actions — I think it's the kind of thing that should be high priority in nature. But it is a good call to observe, and potentially mak